includes/class-sensei-question.php

```diff
@@ -766,64 +765,160 @@ public static function answer_feedback_notes( $question_id ) {
+
+		$grade = Sensei()->view_helper->format_question_points( $answer_grade . '/' . $question_grade );
+
+		/**
+		 * Filter the learner grade displayed.
+		 *
+		 * @hook  sensei_question_answer_message_grade
+		 * @since 3.14
+		 *
+		 * @param {string} $grade          Formatted grade (eg "0/3 points")
+		 * @param {int}    $lesson_id      Lesson ID.
+		 * @param {int}    $question_id    Question ID.
+		 * @param {int}    $user_id        User ID.
+		 * @param {bool}   $answer_correct Whether this is the correct answer.
+		 *
+		 * @return {string} Answer message.
+		 */
+		$grade = apply_filters( 'sensei_question_answer_message_grade', $grade, $lesson_id, $question_id, get_current_user_id(), $answer_correct );
+
+		/**
+		 * Filter the correct answer.
+		 *
+		 * @hook  sensei_question_answer_message_correct_answer
+		 * @since 1.9.0
+		 *
+		 * @param {string} $answer_message Answer message.
+		 * @param {int}    $lesson_id      Lesson ID.
+		 * @param {int}    $question_id    Question ID.
+		 * @param {int}    $user_id        User ID.
+		 * @param {bool}   $answer_correct Whether this is the correct answer.
+		 *
+		 * @return {string} Answer message.
+		 */
+		$correct_answer = apply_filters( 'sensei_question_answer_message_correct_answer', $correct_answer, $lesson_id, $question_id, get_current_user_id(), $answer_correct );
+
+		?>
+		<div class="sensei-lms-question__answer-feedback <?php echo esc_attr( $answer_notes_classname ); ?>">
+			<?php if ( $indicate_incorrect ) { ?>
+				<div class="sensei-lms-question__answer-feedback__header">
+					<span class="sensei-lms-question__answer-feedback__icon"></span>
+					<span
+						class="sensei-lms-question__answer-feedback__title"><?php echo wp_kses_post( $answer_feedback_title ); ?></span>
+					<?php if ( $grade ) { ?>
+						<span class="sensei-lms-question__answer-feedback__points"><?php echo wp_kses_post( $grade ); ?></span>
+					<?php } ?>
+				</div>
+			<?php } ?>
+			<?php if ( $answer_notes || $correct_answer ) { ?>
+				<div class="sensei-lms-question__answer-feedback__content">
+					<?php if ( $correct_answer ) { ?>
+						<div class="sensei-lms-question__answer-feedback__correct-answer">
+							<?php echo wp_kses_post( __( 'Right Answer:', 'sensei-lms' ) ); ?>
+							<strong><?php echo wp_kses_post( $correct_answer ); ?></strong>
+						</div>
+					<?php } ?>
+					<?php if ( $answer_notes && wp_strip_all_tags( $answer_notes ) ) { ?>
+						<div class="sensei-lms-question__answer-feedback__answer-notes">
+							<?php echo wp_kses_post( $answer_notes ); ?>
+						</div>
+					<?php } ?>
+				</div>
+			<?php } ?>
+		</div>
+		<?php if ( $grade ) { ?>
+			<style> .question-title .grade { display: none; } </style>
+		<?php } ?>
+		<?php
+	}
+
+	/**
+	 * Answer feedback.
+	 *
+	 * @param int $question_id Question ID.
+	 */
+	public static function answer_feedback_notes( $question_id ) {
```

Comment:
If I understood correctly, I think it should be deprecated, right?
Also, I noticed there's a `@hooked` comment in `do_action( 'sensei_quiz_question_inside_after', sensei_get_the_question_id() );` that we should update to the `the_answer_feedback`.

Reply:
Update the comment in 5adbc83, and have the deprecated notice in here too.